CONTRIBUTING.md

```diff
@@ -53,6 +53,7 @@ We have a sophisticated CI process setup which gets run on every PR. You must pa
   - You can run `yarn typecheck` in any package or in the root.
 - Ensure your changes are adequately tested.
   - You can run `yarn test` in any package.
+    - [VS Code launch tasks](https://code.visualstudio.com/docs/editor/tasks) tasks are provided that allow [visual debugging](https://code.visualstudio.com/docs/editor/debugging) tests
```

Comment:
@tapanprakasht is this the info you were hoping to see added?

Reply:
Yes, this should be sufficient to debug the code.